airbyte-server/src/main/java/io/airbyte/server/handlers/WebBackendConnectionsHandler.java

```diff
@@ -242,6 +259,26 @@ private WebBackendConnectionListItem buildWebBackendConnectionListItem(
     return listItem;
   }
 
+  @VisibleForTesting
+  SchemaChange getSchemaChange(
```

Comment:
@alovew I refactor this method in order to make it easier to understand. I also added missing test to it. Please make sure that I didn't miss anything in the re-implementation.